.travis.yml

```diff
@@ -12,13 +12,13 @@ jobs:
   - python: '2.7'
     env: TOXENV=py27
   - python: '3.5'
-    env: TOXENV=py35
+    env: TOXENV=py
```

Comment:
No need to explicitly specify `TOXENV` unless there are specific overrides relating to the env.

Reply:
I prefer passing the full TOXENV name for better visual consistency.

Also, this change makes the Travis display a bit awkward:

<img width="200" alt="image" src="https://user-images.githubusercontent.com/2379650/47692521-60e17600-dbcc-11e8-978a-dc96ddff4253.png">